test-commons/src/main/java/no/unit/nva/publication/PublicationGenerator.java

```diff
@@ -234,14 +234,14 @@ private static FileSet sampleFileSet() {
 
     private static License randomLicense() {
         return new License.Builder()
-            .withIdentifier(randomString())
+            .withIdentifier(FAKER.lorem().word())
```

Comment:
I would advise against using Faker.bla.bla, direcly because in refactorings we will have to change stuff in many places. Use randomString here. or crate a method for randomWord.

Reply:
created randomWord as we need something without space